onboard/catkin_ws/src/task_planning/scripts/task_runner.py

```diff
@@ -1,23 +1,33 @@
 #!/usr/bin/env python3
 
 import rospy
-
-from competition_task import CompetitionTask
+import smach.StateMachine
```

Comment:
A different version is in master??? Very confused at this point. Might be worth merging the changes here

Reply:
Merge